TSClientGen.Core/ModuleGenerators/IndentedStringBuilder.cs

```diff
@@ -39,17 +39,17 @@ public IIndentedStringBuilder AppendLine(string text = null)
 		{
 			writeIndentingIfNeeded();
 			if (text != null)
-				_result.AppendLine(text);
+				_result.Append(text).Append("\r\n");
 			else
-				_result.AppendLine();
+				_result.Append("\r\n");
 			
 			_shouldWriteIndenting = true;
 			return this;
 		}
 
 		public IIndentedStringBuilder AppendText(string text)
 		{
-			var lines = text.Split(new[] {Environment.NewLine}, StringSplitOptions.RemoveEmptyEntries);
+			var lines = text.Split(new[] {"\r\n"}, StringSplitOptions.RemoveEmptyEntries);

```

Comment:
It should be more robust to split by `new [] { "\r", "\n" }, StringSplitOptions.RemoveEmptyEntries`

Reply:
I assume that all new lines are `\r\n`, so it should be ok this way. But I can make it more clear and assign `\r\n` to a global static variable. Would that be ok?

Reply:
If this assumption is validated by tests, it should be ok too

Reply:
It is. I actually ran into this problem with line endings while running the tests. Some test was expecting `\n` (the line ending was hard-coded in the file), but it was `\r\n`, since the tests ran under Windows.